css/keyui.sass

```diff
@@ -140,6 +140,14 @@
         border: 1px solid darken($n, 15%)
         background-color: lighten($n, 15%)
         box-shadow: 0 0px 3px darken($n, 10%) inset
+        &:valid
```

Comment:
I like the concept of using HTML5 :valid pseudoclass but unfortunatelly it causes some nasty behavior:
https://stackoverflow.com/questions/27021801/inputinvalid-css-rule-is-applied-on-page-load

By default all elements are invalid.

I think applying .valid and .invalid class is better but it collides with the idea with not using class names :/ We can stay with it for now. 

If you don't use $n argument (which in this case is background color) for contrast calculation move this implementation into %input2

Reply:
It's almost same as mixin but it doesn't take any argument. There are some performance differences under hood aswell. 

https://sass-lang.com/documentation/at-rules/extend